### PR TITLE
[crypto] Add return type for randomUUID

### PR DESCRIPTION
**Pull request by KiwiKilian**
# Why

The return type of `Crypto.randomUUID()` is currently `any`, because no explicit return type is set.

# How

Return type `string` added.

# Test Plan

–

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Comment by KiwiKilian** [comment]
Shall this be included in the changelog? Is it a breaking change?

**Comment by Simek** [comment]
We do not consider type corrections as a breaking change, changelog entry in this case probably should be placed under "Other" category.

Additionally, it would be great if you can regenerate the data for docs by running:
```
et gdad -p expo-crypto
```

**Comment by KiwiKilian** [comment]
Done!

**Comment by Simek** [comment]
Thank you! 👍

It looks like the regenerated JSON has not been minified, have you used the flag to control the output form manually, or it is what the command exports without any alterations/options provided on your machine?

**Comment by KiwiKilian** [comment]
Sorry, my error. I first struggled with the ruby setup and unminified to check if the change looks right. Now I've committed the generated output from `et gdad -p expo-crypto`.

File: packages/expo-crypto/CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
 ### 💡 Others
 
+- Change return type of `randomUUID` to `string`. ([#21187](https://github.com/expo/expo/pull/21187) by [@KiwiKilian](https://github.com/KiwiKilian))
```
Comment (Simek) [inline packages/expo-crypto/CHANGELOG.md:13]:
```suggestion
- Set the missing return type of `randomUUID` method. ([#21187](https://github.com/expo/expo/pull/21187) by [@KiwiKilian](https://github.com/KiwiKilian))
```

Changelog entry phrasing nit.

**Review by Simek — APPROVED** [review]
LGTM, thank you for updates and working on this correction! 👍

**Comment by MrLoh** [comment]
When will this be released, it seems like `12.2.1` still has this issue

**Comment by Simek** [comment]
We are not able to release packages on every change that was pushed, so there are no date set, please watch out the package CHANGELOG:
* https://github.com/expo/expo/blob/main/packages/expo-crypto/CHANGELOG.md

**Comment by MrLoh** [comment]
That's reasonable, I was just surprised since this merged before the 48 release, but I guess it just didn't make it into the release branch. Looking forward to seeing the change in SDK 49. Thank you for all the amazing work that goes into expo.

**Pull request merged**